example-testng-fork-execution/src/main/resources/reportportal.properties

```diff
@@ -0,0 +1,11 @@
+rp.endpoint =
+
+rp.uuid = 
+
+rp.launch = superadmin_TEST_EXAMPLE
+
+rp.project = superadmin_personal
+rp.reporting.async=true
+rp.client.join=true
+rp.client.join.lock.file.name=/Users/yumfriez/qwe/rp.lock
+rp.client.join.sync.file.name=/Users/yumfriez/qwe/rp.sync
```

Comment:
/Users/yumfriez/qwe/ ???
Can we use some file inside resources and relative path to it?

Reply:
+1 this is absolutely unnecessarily to put full path. I beleive these files will be created in build directory if just file name specified.

Reply:
fixed